src/main/resources/templates/Index.html

```diff
@@ -1,85 +1,87 @@
 <!DOCTYPE html>
-<html xml:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" lang="en-US">
```

Comment:
XML Namespace for ThymeLeaf must be defined using `xmlns:th` not `xml:th`. [Reference](https://developer.mozilla.org/en-US/docs/Archive/Mozilla/XUL/Namespaces)